docs/migration/5_0-generated-typescript-changes-doc.md

```diff
@@ -22052,6 +22324,31 @@ Function keyboardFocusFactory has been removed and is no longer part of the publ
 ## @spartacus/storefront
 
 
+### Method closeDialog changed.
+
+
+Previous version: 
+
+```
+
+closeDialog(
+  reason: string
+): void
+
+```
+
+
+Current version: 
+
+```
+
+closeDialog(
+  reason: any
```

Comment:
`reason` is changed from `string` to `any`. I don't think it is breaking change.

Reply:
I agree.  I will remove it.